Installation/changes.html

```diff
@@ -230,6 +230,14 @@ <h2 id="release4.7">Release 4.7 </h2>
     </li>
   </ul>
 
+  
+  <h3>Polyhedron Demo (Scale Space Reconstruction Plugin)</h3>
+  <ul>
+    <li>Speed up construction of polygon soups by only inserting the
+      vertices that are actually used by each polygon soup.
+    </li>
+  </ul>
+
 
```

Comment:
Do we ask Simon to remove the modification to changes.html, or it is acceptable that way?